docs/getting_started.md

```diff
@@ -12,8 +12,9 @@ The required versions of MMCV, MMDetection and MMSegmentation for different vers
 
 | MMDetection3D version | MMDetection version | MMSegmentation version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.12.0          | mmseg>=0.14.1 | mmcv-full>=1.3.2, <=1.4|
-| 0.14.0              | mmdet>=2.10.0, <=2.11.0| mmseg>=0.13.0 | mmcv-full>=1.3.1, <=1.4|
+| master              | mmdet>=2.14.0| mmseg==0.14.1 | mmcv-full>=1.3.1, <=1.4|
```

Comment:
Why is mmcv-full>=1.3.8 [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/__init__.py#L20)?